src/tests/files/brp.json

```diff
@@ -0,0 +1,184 @@
+        "properties": {
+          "geslachtsaanduiding": {
+            "type": "string"
+          },
+          "naam": {
+            "type": "object",
+            "properties": {
+              "aanduidingNaamgebruik": {
+                "type": "string"
+              },
+              "voornamen": {
+                "type": "string"
+              },
+              "voorletters": {
+                "type": "string"
+              },
+              "geslachtsnaam": {
+                "type": "string"
+              },
+              "voorvoegsel": {
+                "type": "string"
+              }
+            },
+            "required": [
+              "aanduidingNaamgebruik",
+              "voornamen",
+              "voorletters",
+              "geslachtsnaam",
+              "voorvoegsel"
+            ]
+          },
+          "leeftijd": {
+            "type": "integer"
+          },
+          "burgerservicenummer": {
+            "type": "string"
+          },
+          "geboorte": {
+            "type": "object",
+            "properties": {
+              "datum": {
+                "type": "object",
+                "properties": {
+                  "datum": {
+                    "type": "string"
+                  },
+                  "jaar": {
+                    "type": "integer"
+                  },
+                  "maand": {
+                    "type": "integer"
+                  },
+                  "dag": {
+                    "type": "integer"
+                  }
+                },
+                "required": [
+                  "datum",
+                  "jaar",
+                  "maand",
+                  "dag"
+                ]
+              }
+            },
+            "required": [
+              "datum"
+            ]
+          },
+          "verblijfplaats": {
+            "type": "object",
+            "properties": {
+              "functieAdres": {
+                "type": "string"
+              },
+              "huisnummer": {
+                "type": "string"
+              },
+              "postcode": {
+                "type": "string"
+              },
+              "straatnaam": {
+                "type": "string"
+              },
+              "datumAanvangAdreshouding": {
+                "type": "object",
+                "properties": {
+                  "datum": {
+                    "type": "string"
+                  },
+                  "jaar": {
+                    "type": "integer"
+                  },
+                  "maand": {
+                    "type": "integer"
+                  },
+                  "dag": {
+                    "type": "integer"
+                  }
+                },
+                "required": [
+                  "datum",
+                  "jaar",
+                  "maand",
+                  "dag"
+                ]
+              },
+              "datumInschrijvingInGemeente": {
```

Comment:
In het schema doen we toch alleen space-separated? Of is dat hier technisch lastig/onmogelijk?

Reply:
Goede vraag: ik heb het BRP schema gegenereerd op basis van wat de remote server teruggeeft. Dat is ook hoe ik letterlijk de veldnamen valideer.

We kunnen hier spaties toevoegen, alleen kan de code niet raden of je dan camelCase van de server verwacht, of dat de server dan in underscores aanlevert.